pkg/apis/build/v1alpha2/image_conversion.go

```diff
@@ -46,8 +45,14 @@ func (is *ImageSpec) convertTo(to *v1alpha1.ImageSpec) {
 	to.SuccessBuildHistoryLimit = is.SuccessBuildHistoryLimit
 	to.ImageTaggingStrategy = is.ImageTaggingStrategy
 	to.Source = is.Source
-	to.Build = is.Build
 	to.Notary = is.Notary
+
+	if fromBuild := is.Build; fromBuild != nil {
```

Comment:
Why fromBuild?

Reply:
I reused it from the convertFrom func, I'll rename it